my_project/tab_wind/app_wind.py

```diff
@@ -141,10 +142,11 @@ def daily_wind_rose():
         id="tab5-daily-container",
         children=[
             html.Div(
-                children=title_with_tooltip(
+                children=title_with_link(
                     text="Daily Wind Rose",
-                    tooltip_text=None,
+                    tooltip_text="Click here to learn more about Daily wind rose.",
```

Comment:
if we keep the name of the chart we should make sure that it is either always capitalised or no. The previous says "wind" and this one "Daily"